src/admin/messages/kr/cmsadmin.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+return [
+ 	 
+-    'view_index_add_type' => '페이지 유형',
+-    'view_index_type_page' => '페이지',
+-    'view_index_type_module' => '모듈',
+-    'view_index_type_redirect' => '전송',
+-    'view_index_as_draft' => '템플릿으로',
+-    'view_index_as_draft_help' => '새 페이지를 템플릿으로 정의 하시겠습니까?',
+-    'view_index_no' => '아니요',
+-    'view_index_yes' => '네',
+-    'view_index_page_title' => '페이지 제목',
+-    'view_index_page_alias' => 'URL 경로 세그먼트',
+-    'view_index_page_meta_description' => '설명 (구글의 메타설명)',
+-    'view_index_page_nav_container' => '탐색 컨테이너',
+-    'view_index_page_parent_page' => '기본 페이지',
+-    'view_index_page_success' => '페이지가 성공적으로 생성되었습니다.',
+-    'view_index_page_parent_root' => '루트 수준',
+-    'view_index_page_use_draft' => '템플릿을 사용하실 건가요?',
+-    'view_index_page_select_draft' => '템플릿을 선택하실 건가요?',
+-    'view_index_page_layout' => '레이아웃 선택',
+-    'view_index_page_btn_save' => '새로운 페이지 저장하기',
+-    'view_index_module_select' => '모듈 이름',
+-    'view_index_sidebar_new_page' => '페이지 만들기',
+-    'view_index_sidebar_drafts' => '템플릿 관리',
+-    'view_index_sidebar_move' => '실행',
+-    'view_update_drop_blocks' => '이곳에 컨텐츠 블록 놓기',
+-    'view_update_blockcontent' => '컨텐츠 차단',
+-    'view_update_configs' => '선택적 구성',
+-    'view_update_settings' => '설정',
+-    'view_update_btn_save' => '저장하기',
+-    'view_update_btn_cancel' => '중단하기',
+-    'view_update_holder_state_on' => '자리 표시 자 접기',
+-    'view_update_holder_state_off' => '자리 표시 자 펼치기',
+-    'view_update_is_draft_mode' => '초안모드에서 편집',
+-    'view_update_is_homepage' => '홈페이지',
+-    'view_update_properties_title' => '페이지 속성',
+-    'view_update_no_properties_exists' => '이 페이지에는 아직 등록된 속성이 없습니다.',
+-    'view_update_draft_no_lang_error' => 템플릿이 이 언어로 설정되어 있지 않습니다..',
```

Comment:
This will still throw an exception as i have mentioned from travis check at the beginning. Missing `'`.